src/core/server/saved_objects/service/lib/saved_objects_aggs_types/bucket_aggs/index.ts

```diff
@@ -0,0 +1,48 @@
+/*
+ * Licensed to Elasticsearch B.V. under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch B.V. licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import * as rt from 'io-ts';
+
+import { FieldBasicRT } from '../helpers';
+
+export const BucketAggsTypeRt = rt.partial({
```

Comment:
Do you mind adding a TODO with the missing bucket aggregations, as you did for metrics in `metrics_aggs/index.ts `?